src/cmd/config/mock.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+import (
+	"github.com/stretchr/testify/mock"
+	"k8s.io/client-go/rest"
+)
+
+type MockProvider struct {
+	mock.Mock
+}
```

Comment:
```suggestion
type MockProvider mock.Mock
```

Reply:
Results in unresolved reference errors when calling the mock functions, e.g. `Called`.
It's also used like that in the [docs](https://pkg.go.dev/github.com/stretchr/testify/mock)